server/index.js

```diff
@@ -29,4 +35,46 @@ app.get('/shoes/:shoeId/rating', (req, res) => {
   });
 });
 
+//create review
+app.post('/shoes/:shoeId/reviews', (req, res) => {
+  let { shoeId } = req.params;
+  let review = req.body;
+  review.shoe_id = Number(shoeId);
+  controller.addReview(review)
+  .then(data => {
+    res.send(data);
+  })
+  .catch(err => {
+    console.error(err);
+    res.end();
+  });
+});
+
+//delete review
+app.delete('/shoes/reviews/:id', (req, res) => {
+  let { id } = req.params;
+  controller.deleteReview(id)
+  .then(data => {
+    res.send(data);
```

Comment:
What data are you sending on a delete? Maybe send the status code for a successful delete?

Reply:
It is sending an object with a message. example: 
{ "message": "Review was deleted successfully!" }